doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-set-user-task-assignment-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+[id='proc-set-user-task-assignment-strategy_{context}']
+
+= Setting the user task assignment strategy
+
+The user task assignment strategy is used to automatically assign the tasks to a suitable user. The assignment strategy allows more efficient task allocation based on the associated properties, such as potential owners, task priority, and task data. The default user task assignment strategy in Red Hat Process Automation Manager is `org.jbpm.task.assignment.strategy`. You can also explicitly define an assignment strategy for a user task in {CENTRAL}.
+
+.Prerequisites
+
+* You have created a project in {CENTRAL}.
+* You must set the `org.jbpm.task.assignment.enabled` system property to `true`.
+
+.Procedure
+
+. Create a business process.
++
+For more information about creating a business process in {CENTRAL}, see <<design-bus-proc>>.
+
+. Create a user task.
++
+For more information about creating a user task in {CENTRAL}, see <<create-user-task-proc>>.
+
+. In the upper-right corner of the screen, click the *Properties* icon.
+. Expand *Implementation/Execution* and click image:getting-started/btn_assign.png[] below to *Assignments*, to open the *Data I/O* window.
+. Add a data input with the name `AssignmentStrategy`, with the type `String`, and with the constant source, such as the strategy name.
++
+Note that if `AssignmentStrategy` is set as null, then no assignment strategy is used for the task.
```

Comment:
```suggestion
NOTE: If `AssignmentStrategy` is set as null, then no assignment strategy is used for the task.
```
If it is important info for the user then can we convert it into a NOTE instead of starting with Note that?

Reply:
Done